include/zephyr/bluetooth/conn.h

```diff
@@ -597,6 +597,71 @@ int bt_conn_le_set_tx_power_report_enable(struct bt_conn *conn,
 					  bool local_enable,
 					  bool remote_enable);
 
+/** @brief Set remote (peer) transmit power.
+ *
+ *  @param conn			Connection object.
+ *  @param phy			PHY bit number i.e. [1M, 2M, s8, s2] == [1, 2, 3, 4]
+ *  @param delta        Delta to be used by remote to apply to its transmit power.
+ *                      Delta is what we as local ask of the peer, it is not
+ *                      synonymous with the changes peer will (or won't) apply.
+ *                      Furthermore, it can be 0 to inquire tx power setting of remote.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @return -ENOBUFS HCI command buffer is not available.
+ */
+int bt_conn_set_remote_tx_power_level(struct bt_conn *conn,
+					 enum bt_conn_le_tx_power_phy phy, int8_t delta);
+
+/** @brief Set remote (peer) transmit power.
+ *
+ *  @param conn              Connection object.
+ *  @param auto_enable       Enable or Disable controller initiated autonomous
+ *                           LE Power Control Request procedure.
+ *                           Disabled by default.
+ *  @param apr_enable        Enable or Disable APR handling in controller during
+ *                           LE Power Control Request procedure.
+ *                           Disabled by default.
+ *  @param beta              Beta value used for exponential weighted averaging of
+ *                           RSSI in 12-bit fixed point fraction.
```

Comment:
It's not so clear what "12-bit fixed point fraction" means and it's confusing. Can we instead move the text "The beta value used in computation is beta/4096." up here?

Reply:
Did so but it still seems a bit convoluted to me. Any ideas? I think it is most important to understand the gist of this... and anyways when I looked to understand this all books referencing this weren't available for free online, so anyways a bit difficult to find deeper understanding (and the algorithm was created by someone who left Nordic a while back)

Reply:
We can't document it as "Lost knowledge. Author quit." :P It has to mean something to our users.

I recognize `avg[n] = beta * avg[n - 1] + (1 - beta) * rssi[n]`. It's the mathematical form of the [Exponential Moving Average]. I assume `avg[n]` is then compared against the targets and limits.

It's not clear to me what `rssi[n]`. Is it the RX RSSI value observed at LL PDU number 'n'? Or the connection event number 'n'? Or something else?

'APR' is not a well known acronym. 'Acceptable power reduction'? Let's spell it out like 'Acceptable power reduction (APR)'. And what effect does that parameter have here?

`wait_period_ms` is also not clear to me. 

> Can we instead move the text "The beta value used in computation is beta/4096." up here?

> Did so but it still seems a bit convoluted to me. What is the trigger of the wait timeout?

I meant to suggest that we somehow replace the sentence "Beta value used for exponential weighted averaging of RSSI in 12-bit fixed point fraction." or remove it entirely. I think we should try to phrase the documentation in terms of what it means for a user of this API. I.e. What is the effect of various values for this parameter.

[Exponential Moving Average]: https://en.wikipedia.org/wiki/Exponential_smoothing

Reply:
Hi, I tried to make it more readable. I left the math, but renamed changed the beta/4096 to "gamma" in the description, so it is easier to understand: https://github.com/nrfconnect/sdk-zephyr/pull/1376/files#diff-4538e90f4e61dd34802aa1c0a5382dc89d4df6032ec6a3dbbf4eef746ebdd488R642
As per Teams communication I opened a PR in sdk-zephyr directly. If you want me to update this one as well, let me know.